script.js

```diff
@@ -0,0 +1,37 @@
+const cardArray = [
+  {
+    id: 1,
+    name: "powerCard",
+    state: false,
+    "back-image": "assets/powercard.jpg",
+    "front-image": "assets/cardpattern.jpg",
+  },
+];
+
+const containerElement = document.querySelector("#container");
+const gameTitleElement = document.createElement("h1");
+gameTitleElement.classList.add("game-title");
+gameTitleElement.innerHTML = "Memory Card Game";
+containerElement.appendChild(gameTitleElement);
+
+const cardsContainerElement = document.createElement("div");
+cardsContainerElement.classList.add("cards-container");
+containerElement.appendChild(cardsContainerElement);
```

Comment:
I would recommend doing all the above setting up of titles and containers in your HTML markup instead of in Javascript. It will be easier to read the code and understand how they are laid out relative to each other in HTML, and aside from getting a reference to the `.cards-container` element, we don't really need to manipulate these elements in Javascript.